packages/lib/modules/pool/actions/add-liquidity/handlers/ProportionalBoostedUnbalancedAddLiquidityV3.handler.ts

```diff
@@ -0,0 +1,40 @@
+import { AddLiquidityBoostedBuildCallInput, AddLiquidityBoostedV3, Hex } from '@balancer/sdk'
+import { TransactionConfig } from '@repo/lib/modules/web3/contracts/contract.types'
+import { SdkBuildAddLiquidityInput } from '../add-liquidity.types'
+import { BaseProportionalAddLiquidityHandler } from './BaseProportionalAddLiquidity.handler'
+import { constructBaseBuildCallInput } from './add-liquidity.utils'
+
+export class ProportionalBoostedUnbalancedAddLiquidityV3Handler extends BaseProportionalAddLiquidityHandler {
+  public async buildCallData({
```

Comment:
There are some duplication  in this and the other v3 proportional handler but I preferred to keep it explicit for now.